pom.xml

```diff
@@ -5,6 +5,13 @@
     <groupId>org.jboss.qa.noe</groupId>
     <artifactId>noe-core-parent</artifactId>
     <version>0.17.11-SNAPSHOT</version>
+    <dependencies>
```

Comment:
This dependency is in an unusual place.

Reply:
Please move the dependency to the correct place below